README.md

```diff
@@ -1,33 +1,64 @@
-  as a document in one go. E.g., "repeats/proposal.md".
-3. Edit the document content, preferably using
-  [Markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).
-4. In the "Commit new file" section, give the draft proposal a name, e.g., "New proposal on repeats."
-5. Optionally add an extended description.
-6. Submit the new proposal by clicking the "Commit new file" button.
+## Committee Members
 
-### Edit proposal
+| Name | Term |
+|---|---|
+| [Johan den Dunnen](https://www.universiteitleiden.nl/en/staffmembers/johan-den-dunnen) (Nederland) | 2013 - |
+| [Marina DiStefano](https://www.linkedin.com/in/marina-distefano/) (United States) | 2022 - 2026 |
+ [Peter Freeman](https://research.manchester.ac.uk/en/persons/peter.j.freeman) (United Kingdom) | 2018 - |
+| [Reece Hart](https://linkedin.com/in/reece) (United States) | 2013 - |
+| [Ros Hastings](https://www.researchgate.net/profile/Ros-Hastings)† (United Kingdom) | 2021 - |
+| [Jeroen Laros](https://github.com/jfjlaros) (Nederland) | 2022 - 2026 |
+| [Terence Murphy]() (United States) | 2021 - 2026 |
+| [Rachel Taylor](https://www.linkedin.com/in/rachel-taylor-phd-90a57099/) (United Kingdom) | 2022 - 2026 |
+| [Alex Wagner](https://www.linkedin.com/in/alexphd/) (United States) | 2022 - 2026 |
 
-1. Navigate to the proposal folder and click on the proposal document name.
-2. Click on the pencil button to edit the document.
-3. In the "Commit changes" section, give a short description of the changes.
-4. Optionally add an extended description.
-5. Submit the changes by clicking the "Commit changes" button.
+† ISCN representative
 
+### Past Members
 
-## Discussions
+- 2013 - 2022
+  - Peter Taschner (Nederland)
+- 2014 - 2021
+  - Anne-Francoise Roux (France)
+- 2020 - 2021
+  - Oleg Agofonov (Norway)
+- 2013 - 2021
+  - Donna Maglott (United States)
+  - ISCN representative - Jean McGowan-Jordan (United States)
+  - HUGO representative - Stylianos Antonarakis (Switzerland)
+  - HGVS representative - Marc Greenblatt (United States)
+  - HVP / GV representative - Amy McAllister (United Kingdom)
+- 2013 - 2018
+  - HVP / GV representative - Tim Smith (Australia)
+- 2013 - 2014
+  - Els Dequeker (Belgium)
+
+
+## Reference
+
+den Dunnen JT, Dalgleish R, Maglott DR, Hart RK, Greenblatt MS, McGowan-Jordan J, et al. **HGVS Recommendations for the Description of Sequence Variants: 2016 Update.** Hum Mutat. 2016. [pubmed:26931183](https://www.ncbi.nlm.nih.gov/pubmed/26931183) [doi:10.1002/humu.22981](http://dx.doi.org/10.1002/humu.22981)
+
+## A note about our name
+
+We've called ourselves by lots of names, including "VarNomen", "MutNomen", "HGVS Recommendations", "HGVS Guidelines", and even just "HGVS" colloquially.  We recognize that this has caused confusion in the community.  Please call us "HGVS Nomenclature".
```

Comment:
```suggestion
We've called the nomenclature by lots of names, including "VarNomen", "MutNomen", "HGVS Recommendations", "HGVS Guidelines", and even just "HGVS" colloquially.  We recognize that this has caused confusion in the community.  Please refer to our recommendations as the "HGVS Nomenclature".
```

☝️ proposing this change to differentiate us (the HVNC) from our recommendations (the HGVS Nomenclature).